src/.vuepress/components/ExtensionsList.vue

```diff
@@ -31,23 +32,30 @@ export default {
 
 	data() {
 		return {
+			// List of available sources
 			extensions: [],
-			last_id: "",			// id of the last source. Used to removed the last coma
 		};
 	},
 
 	async beforeMount() {
+		// Get the versioning.json file of the repo
 		const { data } = await axios.get(this.$props.url + "/versioning.json");
 		this.$data.extensions = data["sources"];
-		this.$data.last_id = data["sources"][data["sources"].length - 1];
 	},
-methods: {
-		iconUrl(extensionId, fileName) {
-			//https://paperback-ios.github.io/extensions/${extension_id}/includes/${file_name}
-			return this.$props.url + "/" + extensionId + "/includes/" + fileName;
+	methods: {
+		getName(sourceName) {
+			// Return sourceName without the intext badges (18+, Country-Proof...)
+			return sourceName.replace(" (18+)", "").replace(" (Country-Proof)", "").replace("[FR] ", "")
```

Comment:
```suggestion
			return sourceName.replace(/[(|[].+[)|\]]/g, "").trim();
```

Good call. Instead of specifying `\w` and `\d` the above is a more generic `.*` version which should match even special characters

Reply:
Still a small issue...
`"[tag1] text (tag2)".replace(/[(|[].+[)|\]]/g, "").trim()` return `""` but should return `"text"`.

Reply:
What kind of madlad puts a tag in the beginning and the end of a title? 

Give `/[[|(][^(|[]+[\]|)]/g` a try instead.

Reply:
Sirus does (or at least wanted to) 😄

Seems to work pretty well. Thanks!